### ISSUE TITLE
Cannot authenticate with a user - config.username != config.user

### ISSUE THREAD

**Issue by rob256** [Percona-Lab/mongodb_consistent_backup#120]
In `mongodb_consistent_backup/Common/Config.py` you're assigning the `user` to `config.user`, however, the `DB` class (and other places) attempt to access `config.username`.

The most probable fix would be to Config.py:

```
-        parser.add_argument("-u", "--user", dest="user", help="MongoDB Authentication Username (for optional auth)", type=str)
+        parser.add_argument("-u", "--user", dest="username", help="MongoDB Authentication Username (for optional auth)", type=str)
```

**Comment by timvaillancourt** [comment]
Nice catch @rob256, thanks! I will get this fixed in the 1.0.2 release coming shortly

**Comment by timvaillancourt** [comment]
This fix is available in this branch until release: https://github.com/Percona-Lab/mongodb_consistent_backup/tree/1.0.2
